<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thresholdValueDialog</class>
 <widget class="QDialog" name="thresholdValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create 3D Mask</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>466</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="tipLabel">
      <property name="text">
       <string>Make sure the range of threshold value to apply a 3d mask:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="minLabel">
        <property name="text">
         <string>Min threshold:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minThresholdSpinBox"/>
      </item>
      <item>
       <widget class="QSlider" name="minThresholdSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="maxLabel">
        <property name="text">
         <string>Max threshold:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxThresholdSpinBox"/>
      </item>
      <item>
       <widget class="QSlider" name="maxThresholdSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
